_episodes/06-agg.md

```diff
@@ -355,10 +355,7 @@ this query:
 > >
 > > You can confirm this, by executing this code:
 > > ```
-> > SELECT AVG(a) FROM (
-> >     SELECT 1 AS a
-> >     UNION ALL SELECT NULL
-> >     UNION ALL SELECT 5);
+> > SELECT AVG(a) FROM (SELECT 1 AS a UNION SELECT NULL UNION SELECT 5);
```

Comment:
We could use  ~~~ vs ``` to keep the consistency

Reply:
Thanks for noticing! Upon fixing this, we found that it was perhaps better to remove UNION altogether here, since it's only briefly introduced earlier (as a challenge in episode 4: https://github.com/swcarpentry/sql-novice-survey/search?q=union&unscoped_q=union), and is potentially confusing here.

We're a bit unsure if UNION need be introduced at all in this introductory course, but that's perhaps better to discuss in a separate issue. Let us know if you have thoughts on this though.